### ISSUE TITLE
javascripts/css links no longar valid in SilverStripe 4.1

### ISSUE THREAD

**Issue by Martimiz** [unclecheese/silverstripe-gridfield-betterbuttons#175]
SilverStripe 4.1 seems to want javascript/css to live in resources/betterbuttons (fysically public/resources/betterbuttons), unless the path is set from the site root, so this seems to fix that:

_config.php

    use \SilverStripe\Control\Director;

    define('BETTER_BUTTONS_DIR', Director::baseURL() . basename(dirname(__FILE__)));

Happy to write a PR if you think this the right solution for now?

Martine

**Comment by Martimiz** [comment]
For now, in my dev setup, I did a quick and dirty .htaccess fix like this:

    RewriteEngine On
    RewriteCond %{REQUEST_URI} !^/betterbuttons/javascript.*
    RewriteCond %{REQUEST_URI} !^/betterbuttons/css.*
    RewriteCond %{REQUEST_URI} !^/betterbuttons/images.*
    RewriteRule ^(.*)$ public/$1

That got it working again, but obviously modules in the root are not the thing anymore? What would be the best course? Besides better buttons finally becoming part of core :)

Martine

**Comment by derralf** [comment]
that somehow did't work for me, so i just added a symlink in the /public/ folder:
`ln -s ../betterbuttons betterbuttons`

**Comment by Martimiz** [comment]
Yes, you’re right, I ultimately wound up with css/javascript/images in public too - but your symlink is a far nicer workaround for now 😊
